src/SpatialNavigation.ts

```diff
@@ -39,6 +40,8 @@ const DIAGONAL_SLICE_WEIGHT = 1;
  */
 const MAIN_COORDINATE_WEIGHT = 5;
 
+const AUTO_RESTORE_FOCUS_DELAY = 300;
```

Comment:
Is this a good magic value to just keep static, or do you think it might make sense to expose this value somehow, so it could be configured?

Reply:
I'm not sure, this is only happening when you remove some component(s) and it will try to restore the focus with the delay, I think it needs to be short enough so the user won't notice it, but also to have some time for multiple components to unmount so it can accumulate all the debounced calls into one final call

Reply:
Agree - Let's do it like that. ☺️